Computer History Museum Archive.js

```diff
@@ -0,0 +1,589 @@
+				"attachments": [
+					{
+						"mimeType": "application/pdf",
+						"title": "IBM 1401 Programming Systems"
+					},
+					{
+						"title": "IBM 1401 Programming Systems Image 1",
+						"mimeType": "image/png"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102623002",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "A guide to Fortran IV programming",
+				"creators": [],
+				"date": "1972",
+				"abstractNote": "Category: Book\nDescription: Second edition.  Signed by McCracken on title page.\nType: Document",
+				"archive": "Computer History Museum",
+				"archiveLocation": "X3682.2007",
+				"extra": "Identifying Numbers: ISBN10: 0471582816\nLCCN\n72-4745\nLOC call num\nQA76.73.F25 M3 1972\nDimensions: 28 cm.",
+				"libraryCatalog": "102623002",
+				"numPages": "288",
+				"publisher": "John Wiley & Sons; New York",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102790985",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Logic Programming Workshop '83 poster",
+				"creators": [],
+				"date": "1983",
+				"abstractNote": "Category: Promotional Material\nDescription: PDF scan of the poster for the Logic Programming Workshop '83 in Algarve Portugal, June 26 - July 1, 1983.\nType: Document",
+				"archive": "Computer History Museum",
+				"archiveLocation": "X9292.2020",
+				"libraryCatalog": "102790985",
+				"numPages": "1; 0.0004 GB",
+				"attachments": [
+					{
+						"mimeType": "application/pdf",
+						"title": "Logic Programming Workshop '83 poster"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102620354",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "LP20 Linear Programming System",
+				"creators": [],
+				"abstractNote": "Collection Title: 1620 Restoration Project Collection\nCategory: Manual\nType: Document",
+				"archive": "Computer History Museum",
+				"archiveLocation": "X4248.2008",
+				"extra": "Identifying Numbers: Program ID number: 10.1.009\nCredit: Gift of John Maniotes",
+				"libraryCatalog": "102620354",
+				"publisher": "International Business Machines Corporation (IBM)",
+				"attachments": [],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://www.computerhistory.org/collections/catalog/102765924",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Jacquard programming cards",
+				"creators": [],
+				"abstractNote": "Category: I/O/punched card device\nDescription: The object is a series of cardboard punch cards connected by string. The cards are hand numbered sequentially from 6 to 44.\nType: Physical Object",
```

Comment:
Some of this looks like it should be in Extra.

Reply:
fixed. all moved to Extra/